### ISSUE TITLE
Homonymous messages declarated in different other messages

### ISSUE THREAD

**Issue by njspok** [kinow/protobuf-uml-diagram#10]
UML Diagram must separate different 'message' declaration inner other 'messages'.
For example:
```
syntax = "proto3";

service Calc {
    rpc Calc (SomeRequest) returns (Response) {}
    rpc SingleCalc(SomeOtherRequest) returns (Response) {}
}

message SomeRequest {
    message Shipment {
        string key = 1;
        string type = 2;

    }
    repeated Shipment shipments = 1;
}

message SomeOtherRequest {
    message Shipment {
        string key = 1;
        string type = 2;
    }
    repeated Shipment shipments = 1;
}

message Response {
    string code = 1;
}
```
Shipment messages is different entity - SomeRequest:Shipment and SomeOtherRequest:Shipment and on diagram must two elements, but isnt.

![v_ex_pb2](https://user-images.githubusercontent.com/1404034/80571306-d292be00-8a15-11ea-9f6d-cf124e4f5b87.png)

**Comment by kinow** [comment]
Hi @njspok totally agree.

Would this one look better? Got it after updating the code to use `.full_name` instead of just `.name`. Sounds like the simplest fix?

![issue_10_pb2](https://user-images.githubusercontent.com/304786/80576392-37bbd300-8a59-11ea-8108-ffc3a33b9ea4.png)

Thanks
Bruno

**Comment by kinow** [comment]
@njspok I'm cutting a release with the fix in #10. Feel free to test it later and report in case of issues :+1: Thanks!!
